Optimze thanos example (#3726)
pingcap/tidb-operator: examples/monitor-with-thanos/README.md

> # TiDBMonitor with Thanos

Reviewer: ```suggestion
# TidbMonitor with Thanos
```

This document is to show how to integrate TidbMonitor with [Thanos](https://thanos.io/design.md/).

> ## Install

Reviewer: ```suggestion
## Install TidbMonitor
```

The following commands is assumed to be executed in this directory.

Install the monitor with thanos sidecar:

```bash
> kubectl -n <namespace> apply -f tidb-monitor.yaml
```

> Wait for monitor Pods ready:

Reviewer: ```suggestion
Wait for the monitor Pod ready:
```

```bash
watch kubectl -n <namespace> get pod
```

> If you need to store historical data, you can configure the `objectStorageConfig` field and create the corresponding secret.

Reviewer: Currently, the monitor yaml already has `objectStorageConfig` configured, should we add a step to create the secret before creating the monitor? And add a description that users do not have to create the secret and configure this field if no need to store the historical data.

# TiDBMonitor with Thanos

> > **Note:**
> >
> > This document is to show how to integrate tidbmonitor into [thanos](https://thanos.io/design.md/).
> >
> The following steps will create a TiDB cluster by default.

Reviewer: ```suggestion
This document is to show how to integrate TidbMonitor with [Thanos](https://thanos.io/design.md/).
```

## Install

The following commands is assumed to be executed in this directory.

Install the monitor with thanos sidecar:

```bash
> kubectl -n <namespace> apply -f tidb-monitor.yaml
```

Wait for monitor Pods ready:

```bash
watch kubectl -n <namespace> get pod
```

> If you need to store historical data,you can spec `objectStorageConfig` field and create the corresponding secret.

Reviewer: ```suggestion
If you need to store historical data, you can configure the `objectStorageConfig` field and create the corresponding secret.
```

## Install Thanos

Install thanos query component to integrate tidbmonitor :

```bash

> > kubectl -n <namespace> apply -f thanos-query.yaml

Reviewer: We should combine the svc and deployment yaml into one file?

Author: Ok.

# TidbMonitor with Thanos

> This document is to show how to integrate TidbMonitor with [Thanos](https://thanos.io/design.md/).

Reviewer 2: https://thanos.io/design.md/ page not found
